homeassistant/components/smarty/strings.json

```diff
@@ -42,6 +42,11 @@
         "name": "Boost state"
       }
     },
+    "button": {
+      "reset_filters_timer": {
+        "name": "Reset filters timer"
```

Comment:
filter or filters?

Reply:
There are two intakes to the VMC device: from outside to inside and from inside to outside. Each one has one filter... so **two filters** in total to change.

Reply:
Even the original library is not consistent on this:

        client.**filter_timer** = 31
        client.**reset_filters_timer**.return_value = True

Maybe to be aligned in another PR

Reply:
It just sounds strange and not very natural, would it make sense to name it `Reset filter timers`?

Reply:
To be precise: the filters are two, the timer is one...